### DISCUSSION TITLE
Svelte 5 support

### DISCUSSION THREAD

**Discussion by development-vargamarcel** [aklinker1/vite-plugin-web-extension#207] [General]
How could I implement svelte 5 support?
Tried and failed.

**Comment by aklinker1** [comment]
Since it's not out yet, I don't know, sorry.

**Reply by development-vargamarcel** [reply]
Thank you for the fast reply.
It appears I was overly fatigued during my initial attempt, as the solution is quite straightforward.


For anyone needing this, here is the migration commit: https://github.com/development-vargamarcel/webautomatorkit-vpwe/commit/49f124fceca755557dd61bcaba1b7142d004d92a
